ide/coqide/coqOps.ml

```diff
@@ -903,12 +902,9 @@ object(self)
     Coq.bind (Coq.return ()) (fun () ->
     messages#default_route#clear;
     let point = self#get_insert in
-    if point#compare self#get_start_of_input >= 0 then
-      self#process_until_iter point
-    else begin
-      self#scroll_to_start_of_input ();
-      self#backtrack_to_iter ~move_insert:false point
-    end)
+    if point#compare self#get_start_of_input >= 0
+    then self#process_until_iter point
+    else self#backtrack_to_iter ~move_insert:false point)
```

Comment:
You like formatting the statement this way?

Reply:
I reverted to the previous state for diff cleanliness purposes. Otherwise I do tend to prefer having `then` on the same line as `if` indeed.